http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/common/Constants.java

```diff
@@ -253,10 +264,36 @@
     public static final String HTTP_SOURCE_HANDLER = "SourceHandler";
     public static final String WEBSOCKET_SOURCE_HANDLER = "ws_handler";
     public static final String TARGET_HANDLER = "targetHandler";
+    public static final String REDIRECT_HANDLER = "redirectHandler";
     public static final String IDLE_STATE_HANDLER = "idleStateHandler";
     public static final String HTTP_TRACE_LOG_HANDLER = "http-trace-logger";
     public static final String WEBSOCKET_SERVER_HANDSHAKE_HANDLER = "websocket-server-handshake-handler";
 
+    public static final AttributeKey<Integer> REDIRECT_COUNT = AttributeKey.<Integer>valueOf
+            ("REDIRECT_COUNT");
+    public static final AttributeKey<HTTPCarbonMessage> ORIGINAL_REQUEST = AttributeKey.<HTTPCarbonMessage>valueOf
+            ("ORIGINAL_REQUEST");
+    public static final AttributeKey<HttpResponseFuture> RESPONSE_FUTURE_OF_ORIGINAL_CHANNEL = AttributeKey
+            .<HttpResponseFuture>valueOf
+            ("RESPONSE_FUTURE_OF_ORIGINAL_CHANNEL");
```

Comment:
Shall we append this line with the above?

Reply:
Fixed this. https://github.com/wso2/carbon-transports/pull/412/files#diff-9e2df4e3d298525a473d5d10e8c5d5e9R272